### PR TITLE
PS-7955 Improve zenfs build

### PR DESCRIPTION
**Pull request by EvgeniyPatlan**

### CODE REVIEW

File: build-ps/debian/rules [hunk 1]
```diff
@@ -34,6 +34,7 @@ DEB_HOST_MULTIARCH ?= $(shell dpkg-architecture -qDEB_HOST_MULTIARCH)
 TOKUDB_OPTS_DEFAULT ?= -DWITH_VALGRIND=OFF -DUSE_VALGRIND=OFF -DDEBUG_EXTNAME=OFF -DBUILD_TESTING=OFF -DUSE_GTAGS=OFF -DUSE_CTAGS=OFF -DUSE_ETAGS=OFF -DUSE_CSCOPE=OFF -DTOKUDB_BACKUP_PLUGIN_VERSION=$(TOKUDB_BACKUP_VERSION)
 TOKUDB_OPTS_RELEASE ?= $(TOKUDB_OPTS_DEFAULT) -DTOKU_DEBUG_PARANOID=OFF
 TOKUDB_OPTS_DEBUG ?= $(TOKUDB_OPTS_DEFAULT) -DTOKU_DEBUG_PARANOID=ON
+ZENFS_OPTS_RELEASE ?= -DROCKSDB_PLUGINS=zenfs -DWITH_ZENFS_UTILITY=ON -DWITH_ZBD=bundled
```
Comment (percona-ysorokin) [inline build-ps/debian/rules]: May be `ZENFS_OPTS_DEFAULT` would be a better name here. `RELEASE` is a bit confusing here.

File: build-ps/percona-server-8.0_builder.sh [hunk 2]
```diff
@@ -468,19 +463,13 @@ install_deps() {
         else
             apt-get -y install libzstd1-dev
         fi
+        if [[ ${DIST} == 'focal' ]] || [[ ${DIST} == 'hirsute' ]] || [[ ${DIST} == 'bullseye' ]]; then
+            apt-get -y install libgflags-dev util-linux
```
Comment (percona-ysorokin) [inline build-ps/percona-server-8.0_builder.sh]: `util-linux` is not actually needed here. It was used only on Jenkins to get `blkzone` utility to get summary of `/dev/nullbX` devices.

File: build-ps/debian/rules (hunk 1 above)
Comment (EvgeniyPatlan) [reply build-ps/debian/rules]: fixed

File: build-ps/percona-server-8.0_builder.sh (hunk 2 above)
Comment (EvgeniyPatlan) [reply build-ps/percona-server-8.0_builder.sh]: fixed

**Review by vorsel — APPROVED** [review]

**Review by percona-ysorokin — APPROVED** [review]
LGTM

**Pull request merged**